index.html

```diff
@@ -10,8 +10,190 @@
       href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
       rel="stylesheet"
     />
-    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
-    <title>Creative Commons Global Summit 2015</title>
+
+    <link rel="stylesheet" href="style.css" />
+    <title>Europe Global Summit 2023</title>
   </head>
-  <body></body>
+  <body>
+    <section class="main">
+      <div class="mobile-menu">
+        <img
+          src="./images/ClosingIcon.png"
+          alt="closing icon"
+          class="closing-icon"
+        />
+        <a href="index.html" class="logo-imgs"
+          ><img src="images/earth-globe.png" class="mobile-logo" alt="logo"
+        /></a>
+        <ul class="navigation">
+          <li class="menu"><a href="about.html" class="hover">About</a></li>
+          <li class="menu"><a href="#program" class="hover">Program</a></li>
+          <li class="menu"><a href="#join" class="hover">Join</a></li>
+          <li class="menu"><a href="#sponsor" class="hover">Sponsor</a></li>
+          <li class="menu"><a href="#news" class="hover">News</a></li>
+        </ul>
+      </div>
+      <div class="nav-left">
+        <img src="images/enabled.jpg" alt="Enable icon" class="enabled-menu" />
+      </div>
+      <header class="desktop-menu">
+        <div class="pre-header">
+          <div class="pre-header-flex">
+            <ion-icon name="logo-facebook" class="icon"></ion-icon>
+            <ion-icon name="logo-twitter" class="icon"></ion-icon>
+            <span class="tabs">English</span>
+            <span class="tabs">Language</span>
+            <span class="tabs">Logout</span>
+          </div>
+        </div>
+        <div class="header">
```

Comment:
- [ ] You did a fantastic job creating your navbar :1st_place_medal:, but it is required by the template to align the list of page links to the right :+1:, for all your pages

**Your design**

![Screenshot from 2023-02-07 07-54-02](https://user-images.githubusercontent.com/105117832/217250922-1384dde8-b835-419b-9dc8-ad4d943e7721.png)

**Template**

![Screenshot from 2023-02-07 07-57-17](https://user-images.githubusercontent.com/105117832/217251136-b1894ffa-7ead-4764-a368-67ca3a1bfb15.png)